interface/login/login.php

```diff
@@ -30,6 +30,50 @@
 use OpenEMR\Core\Header;
 $ignoreAuth=true;
 require_once("../globals.php");
+
+// mdsupport - Add 'App' functionality for user interfaces without standard menu and frames
+// If this script is called with app parameter, validate it without showing other apps.
+//
+// Build a list of valid entries
+$rs = sqlStatement ( "SELECT option_id, title,is_default FROM list_options
+		WHERE list_id=? and activity=1 ORDER BY seq, option_id",
+		array ('Apps') );
+$emr_app = array();
+if (sqlNumRows($rs)) {
+	while ( $app = sqlFetchArray ($rs) ) {
+		$app_req = explode ( '?', trim($app['title']) );
+		if (! file_exists ( '../'.$app_req[0]))
+			continue;
+			$emr_app [trim ( $app ['option_id'] )] = trim ( $app ['title'] );
+			if ($app ['is_default'])
+				$emr_app_def = $app ['option_id'];
+	}
+}
+$div_app = '';
+if (count($emr_app)) {
+	// Standard app must exist
+	$std_app = 'main/main_screen.php';
+	if (!in_array($std_app, $emr_app)) {
+		$emr_app['*OpenEMR'] = $std_app;
+	}
```

Comment:
I initially thought it would be best to just hard-code OpenEMR in, but is nice to have it in list_options also in case user wishes to change the name of this option or even redirect it to somewhere else.